src/components/2022/logo.js

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+
+const Logo = () => {
+  return (
```

Comment:
NitPick ~ maybe a more descriptive name so we know what Logo "Type" the component is without having to look at the component itself? LGTM though!

Reply:
Fair enough. I’m all for making this more clear. I changed it to `SparkboxLogo`.